daemon/cluster/cluster.go

```diff
@@ -1367,7 +1363,7 @@ func (c *Cluster) CreateNetwork(s apitypes.NetworkCreateRequest) (string, error)
 	defer c.RUnlock()
 
 	if !c.isActiveManager() {
-		return "", c.errNoManager()
+		return "", errors.NewRequestNotAcceptableError(c.errNoManager())
```

Comment:
Thanks.
While I think 421 is not used so frequently, and in pkg net/http it seems not defined. 
In additional, in this situation client seems no incorrect action, so maybe 4xx is not proper. How about 501(not implemented), since it is the service side condition not meet.